src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int firstSide;
+    private int secondSide;
+
+    public Rectangle(int firstSide, int secondSide, Color color) {
+        super(color);
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+    }
+
+    public void setFirstSide(int firstSide) {

```

Comment:
Setter methods are not necessary if fields are set via constructor and do not change.